index.html

```diff
@@ -226,14 +226,14 @@ <h2 class="footer-title">Contact me</h2>
         </div>
       </div>
     </footer>
-   
```

Comment:
- [ ] When I click on the live demo link in your README, I am redirected to your repository. Deployment is not a requirement for this milestone. If you cannot get the correct live demo link at this time, you can either delete that sub-section or replace the link with the text `Coming Soon ...`

Reply:
Changed the README.md file as you suggested